test/test_populationstrategy.py

```diff
@@ -108,3 +109,13 @@ def test_transitions_not_modified(population_strategy: PopulationStrategy):
                " modified the transitions".format(population_strategy))
 
     assert same, err_msg
+
+   
+def test_ListPopulationSize():
+    pop_size = pyabc.ListPopulationSize(values=[10, 5, 1.5])
+
+    # might be useful to test integration, but for the moment
+    # standalone tests may suffice
+    assert pop_size(0) == 10
+    assert pop_size(2) == 1.5
```

Comment:
I think calling the population size like this is currently not possible. Instead, nr_particles must be directly used.

Reply:
It is actually planned to have a population size ``__call__`` function which would do exactly this here, which replaces the ``nr_particles``. If you would like to do that for all population strategies, please go ahead, otherwise it would be enough to just make the test run here.

Reply:
Sorry, this was a simple copy paste without thinking... I removed the assertions now, so the test should work.
To be honest I do not understand what you mean with the implementation of a __call__ function so I would leave it with this for now.